app/Modules/Newsletter/Console/RenderNewslettersCommand.php

```diff
@@ -10,12 +10,12 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 
-class CreateNewslettersCommand extends Command
+class RenderNewslettersCommand extends Command
 {
 	protected function configure()
 	{
-		$this->setName('newsletters:create')
-			->setDescription('Create newsletters');
+		$this->setName('newsletters:render')
+			->setDescription('Render users\' newsletters prepared to send');
```

Comment:
I recommend avoiding this quota plays in any message if possible. Rephrase or use sprintf.
It's difficult to spot and can make some mess in output.